pkg/kubelet/kubelet.go

```diff
@@ -1633,7 +1634,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		return err
 	}
 	if err := kl.volumeManager.WaitForAttachAndMount(defaultedPod); err != nil {
-		kl.recorder.Eventf(pod, api.EventTypeWarning, events.FailedMountVolume, "Unable to mount volumes for pod %q: %v", format.Pod(pod), err)
```

Comment:
Let's leave this as is. It is a catch all event. If there is a failure in the storage stack (some where where operation executor doesn't generate an event) that results in a volume not getting mounted, this at least tells the user where the issue is.